gufe/protocols/protocol.py

```diff
@@ -214,6 +215,14 @@ def create(
             A directed, acyclic graph that can be executed by a `Scheduler`.
 
         """
+        # coerce mapping argument into list for _create signature
+        # top level "create" allows more intuitive input
+        # internal "_create" provides guaranteed type
+        if mapping is None:
```

Comment:
Both the AFE and MD protocols expect `None` (although neither checks for it right now).

In practice there will be more Protocols without mappings than Protocols with mappings - i.e. it feels like we're bending things to be purely RFE orientated when it would make a lot more sense (especially in a Pythonic manner) for a protocol author to expect `None` rather than `empty list`